MediaGalleryIntegration/Plugin/UpdateWysiwygOpenDialogUrl.php

```diff
@@ -60,14 +60,15 @@ public function afterGetConfig(DefaultConfigProvider $provider, DataObject $conf
             return $config;
         }
 
-        return $config->setData(
-            'files_browser_window_url',
-            $this->url->getUrl(
-                'media_gallery/index/index',
-                [
-                    'current_tree_path' => $this->imagesHelper->idEncode(Config::IMAGE_DIRECTORY)
-                ]
-            )
+        $currentTreePath = $this->imagesHelper->idEncode(Config::IMAGE_DIRECTORY);
+        $filesBrowserWindowUrl = $this->url->getUrl(
+            'media_gallery/index/index',
+            [
+                'current_tree_path' => $currentTreePath,
+            ]
         );
+        $config->setData('files_browser_window_url', $filesBrowserWindowUrl);
+
+        return $config;
```

Comment:
Why?) It was more readable

Reply:
I followed the next principle: move all data manipulations outside the `setData` method to avoid data creation in method parameters :)

Reply:
Where it is derived from?

Reply:
It is only my personal representation. I think that it is not so clear to have aa more than one operation in the data set. Be came more complicated to read when you see a code at first :)

Reply:
I think the existing code was better readable too. Also, I'd avoid creating variables for a single usage unless it's absolutely necessary. Can you please revert these changes